app/components/question_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class QuestionComponent < ApplicationComponent
+  include ApplicationHelper
+  include BootstrapHelper
+  include UserHelper
+
+  def initialize(question:, context_user: nil, collapse: true, hide_avatar: false, profile_question: false)
+    @question = question
+    @context_user = context_user
+    @collapse = true
```

Comment:
oh yeah rubogarda is right about this one

```suggestion
    @collapse = collapse
```

I wish Ruby had something like Crystal's initialize syntax, where all this wouldn't be necessary:
```crystal
def initialize(@question : Question, @context_user : User?, @collapse = true, @hide_avatar = false, @profile_question = false)
```